Rakefile

```diff
@@ -194,4 +219,50 @@ end
 def cprint(string)
   print "\033[1m#{string}\033[0m"
 end
+
+def retrieve_vm_as_ova(vmname)
+  ovaname = "puppet-#{VMTYPE}vm-#{PEVERSION}-#{RELEASE}"
+  vcenter_config = File.join(CACHEDIR, ".vmwarecfg.yml") || ENV["VCENTER_CONFIG"]
+  vcenter_settings = YAML::load(File.open(vcenter_config))
+  FileUtils.rm_rf(OVFDIR) if File.directory?(OVFDIR)
+  FileUtils.mkdir_p(OVFDIR)
+  Dir.chdir(BUILDDIR) do
+    verbose(false) do
+      sh(%Q</usr/bin/ovftool --noSSLVerify --targetType=OVA --compress=9 --name=#{ovaname} --powerOffSource vi://#{vcenter_settings['username']}\\@puppetlabs.com:#{vcenter_settings['password']}@vmware-vc2.ops.puppetlabs.net/opdx2/vm/vmpooler/centos-6-i386/#{vmname}  #{OVFDIR}/>)
+    end
+  end
+  File.join(OVFDIR, ovaname) + ".ova"
+end
+
+def ship_vm_to_vmware(vmpath)
+  require 'rbvmomi'
+  vcenter_config = File.join(CACHEDIR, ".vmwarecfg.yml") || ENV["VCENTER_CONFIG"]
+  vcenter_settings = YAML::load(File.open(vcenter_config))
```

Comment:
Yes, that config file is specific to our jenkins slaves and the shipping tasks, so I didn't want that to leak into the environment if someone is running this locally. I doesn't really matter, as long as they don't try to use the values, but I was mostly just trying to keep it local to the methods that use it.